config.cfg

```diff
@@ -74,9 +74,10 @@ congrats: |
   "#    Config files and certificates are in the ./configs/ directory.    #"
   "#              Go to https://whoer.net/ after connecting               #"
   "#        and ensure that all your traffic passes through the VPN.      #"
-  "#          Local DNS resolver and Proxy IP address: {{ local_service_ip }}     "
-  "#                     The p12 password is {{ easyrsa_p12_export_password }}      "
-  "#                  The CA key password is {{ easyrsa_CA_password }}      "
+  "#          Local DNS resolver and Proxy IP address: {{ local_service_ip }}         #"
+  "#                     The p12 password is {{ easyrsa_p12_export_password }}                     #"
+  "#                  The CA key password is {{ easyrsa_CA_password }}                 #"
+  "#            Shell access: ssh -i algo.pem root@{{ ansible_ssh_host }}          #"
```

Comment:
I was assuming the reader had already cd'd to the configs directory, but super minor either way.